spec/spec_helper.rb

```diff
@@ -8,6 +8,8 @@ def horizontal_rule(width = 5)
 
 require 'puppetlabs_spec_helper/module_spec_helper'
 
+FUTURE_PARSER = ENV['FUTURE_PARSER'] == 'true' || ENV['FUTURE_PARSER'] == 'yes'
+
```

Comment:
The testing of Puppet 3.x with future_parser is coming from modulesync. The affected tests are passing through a number instead of a string.  As future parser is type aware this is expected behaviour.  So I can either write the test to always pass a string, or write the test to always treat numbers as an invalid string, or not test with future parser turned on.  Thoughts?